network-api/app/networkapi/views.py

```diff
@@ -0,0 +1,13 @@
+from django.http import JsonResponse
+from django.views import View
+from mezzanine.conf import settings
+
+
+class EnvVariablesView(View):
+    """
+    A view that permits a GET to expose whitelisted environment
+    variables in JSON.
+    """
+
+    def get(self):
```

Comment:
I get an error here when running this: `get() takes 1 positional argument but 2 were given`